### PR TITLE
Pull secret distributor

### PR DESCRIPTION
**Pull request by rpahli**
#### What this PR does / why we need it:

adds the helm chart for: https://github.com/kiwigrid/pull-secret-distributor
this tool distibutes pull secretes across namespaces

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://developercertificate.org) signed
- [x] Chart Version bumped (if the pr is an update to an existing chart)
- [x] Variables are documented in the README.md

### CODE REVIEW

**Comment by monotek** [comment]
I'd like to see a config option to exclude one or more namespaces.

File: charts/pull-secret-distributor/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+appVersion: "1.0"
```
Comment (axdotl) [inline charts/pull-secret-distributor/Chart.yaml]: image.tag does not match 1.0

File: charts/pull-secret-distributor/templates/NOTES.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+Deployments successful installed.
+
+you can check if it's working by creating a new namespace. The pull secrets should be polulated to the new namespace.
```
Comment (axdotl) [inline charts/pull-secret-distributor/templates/NOTES.txt]: typo
poPulated

File: charts/pull-secret-distributor/templates/deployment.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
+apiVersion: apps/v1beta2
```
Comment (axdotl) [inline charts/pull-secret-distributor/templates/deployment.yaml]: Should be apps/v1

File: charts/pull-secret-distributor/values.yaml [hunk 4]
```diff
@@ -0,0 +1,26 @@
+# Default values for pull-secret-distributor.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+image:
+  repository: kiwigrid/pull-secret-distributor
+  tag: 10
+  pullPolicy: IfNotPresent
+# csv list of secrets
```
Comment (axdotl) [inline charts/pull-secret-distributor/values.yaml]: Please provide an example

**Comment by rpahli** [comment]
@monotek please add an issue to the source code repo, THX

**Comment by monotek** [comment]
https://github.com/kiwigrid/pull-secret-distributor/issues/2

**Pull request merged**